package.json

```diff
@@ -1,121 +1,28 @@
 {
-  "name": "@fishjam-dev/react-native-client",
-  "version": "0.3.0",
-  "description": "A React Native client for Membrane WebRTC server.",
-  "main": "build/index.js",
-  "types": "build/index.d.ts",
-  "scripts": {
-    "build": "expo-module build",
-    "clean": "expo-module clean",
-    "format:fix": "prettier --write . --ignore-path ./.eslintignore",
-    "format:check": "prettier --check . --ignore-path ./.eslintignore",
-    "lint:fix": "eslint . --ext .ts,.tsx --fix",
-    "lint:check": "eslint . --ext .ts,.tsx",
-    "test": "expo-module test",
-    "prepare": "expo-module prepare",
-    "prepublishOnly": "expo-module prepublishOnly",
-    "expo-module": "expo-module",
-    "open:ios": "open -a \"Xcode\" example/ios",
-    "open:android": "open -a \"Android Studio\" example/android",
-    "release": "release-it",
-    "releaseAppAndroid": "cd example/android && fastlane deploy",
-    "releaseAppIos": "cd example/ios && fastlane deploy",
-    "docs": "sed 's/\\.github\\/images/media/g' README.md > README_docs.md && typedoc src/index.tsx --media ./.github/images --readme README_docs.md && rm README_docs.md",
-    "typecheck": "yarn tsc --noEmit && cd example && yarn tsc --noEmit"
-  },
-  "keywords": [
-    "react-native",
-    "expo",
-    "@fishjam-dev/react-native-client",
-    "MembraneWebRTC",
-    "Fishjam"
-  ],
-  "repository": "https://github.com/fishjam-dev/react-native-client-sdk",
-  "author": "Software Mansion (https://swmansion.com)",
-  "license": "Apache-2.0",
-  "bugs": {
-    "url": "https://github.com/fishjam-dev/react-native-client-sdk/issues"
-  },
-  "homepage": "https://github.com/fishjam-dev/react-native-client-sdk#readme",
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org/",
-    "access": "public"
-  },
-  "dependencies": {
-    "phoenix": "^1.7.6",
-    "promise-fs": "^2.1.1",
-    "protobufjs": "^7.2.4"
+  "name": "fishjam-mobile-sdk",
+  "version": "0.4.0",
+  "license": "MIT",
+  "private": true,
+  "packageManager": "yarn@4.3.0",
```

Comment:
Does this pr update yarn version?
If so add it to PR description
Such big change should be noted somewhere.

Reply:
In general this PR is big change, as it introduce new project structure. Updating `yarn` is just part of it, to make `yarn workspaces` work fine. But I added command about it to PR description, to make it visible